### PR TITLE
Fix benchmarks

### PR DESCRIPTION
**Pull request by klahnakoski**
Fix for 

https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/535

### CODE REVIEW

**Comment by klahnakoski** [comment]
@davehunt for visual review:

https://deploy-preview-552--firefox-health-dashboard.netlify.com/quantum/64
https://deploy-preview-552--firefox-health-dashboard.netlify.com/quantum/32

**Comment by klahnakoski** [comment]
@davehunt the visual review is all I needed from you.  The diff, is messy because code moved, but the change is simple. Since you are interested, and maybe others too, here is an explanation:

The `<PerfherderWidget` was a D3 chart that charted a specific signature.  Here is an example:

```
      signatures: {
        'windows10-64': '8f014459793e2e94c3244d5edeaada0452b0c627',
      },
```

These signatures changed, either a a platform change or suite name change, or something. so the chart crashed, leaving the red `Object(...)(...) is undefined` error. Instead of hunting down the new signature (only to be foiled again, when something minor changes again), I decided to replace the chart library.

These charts were replaced with `<PerfherderGraphContainer`, which is the chosen charting library for this dashboard.  With that, the signatures are replaced with an explicit expressions, for example:

```
filter: { and: [
        { missing: "test"},
        { eq: {
          suite: "raptor-motionmark-htmlsuite-firefox",
          repo: "mozilla-central",
          framework: 10,
          platform: "windows10-64"
        }}
      ]}
```

With this change, the configuration for the charts was also cleaned up:  Some was in `Quantum.jsx` and some in `config.js`, along with some logic that combined it.  This was all moved to `config.js`, and renamed from `CONFIG` to `BENCHMARKS`.

Since **Speedometer** is in the same **Benchmarks** section, and was already a `<PerfherderGraphContainer`,  it's configuration was moved to the `BENCHMARKS` also.

Overall, you will see a lot of code removed from `Quantum.jsx`; replaced with a simpler expression to show all `BENCHMARKS`.  And you will see a lot of code changed in `config.js` to feed the new `<PerfherderGraphContainer`, plus see the **Speedometer** config added too.

File: src/quantum/Quantum.jsx [hunk 1]
```diff
@@ -568,11 +515,7 @@ export default class QuantumIndex extends React.Component {
           <Grid container spacing={24}>
             {rows.map((widget, wi) => {
               // Acumulate the section's status
-              if (widget.type.displayName !== 'PerfherderWidget') {
-                statusList[widget.props.status] += 1;
-              } else if (widget.props.status === 'red') {
-                statusList.secondary += 1;
-              }
+              statusList[widget.props.status] += 1;
```
Comment (klahnakoski) [inline src/quantum/Quantum.jsx]: @davehunt  We no longer use `PerfherderWidget`, so this is always true.

File: src/quantum/Quantum.jsx [hunk 2]
```diff
@@ -82,72 +81,20 @@ export default class QuantumIndex extends React.Component {
...
-                        framework: 10,
-                        repo: 'mozilla-central',
-                      },
-                    },
-                  ],
-                },
-              },
-            ]}
-          />,
-        ],
```
Comment (klahnakoski) [inline src/quantum/Quantum.jsx]: @davehunt  This config is moving to `confg.js`

File: src/quantum/config.js [hunk 3]
```diff
@@ -7,48 +7,149 @@ import { getSignatures } from "../vendor/perfherder";
...
+      bits: 64,
+      filter: { and: [
+        { missing: "test"},
+        { eq: {
+          suite: "raptor-motionmark-htmlsuite-firefox",
+          repo: "mozilla-central",
+          framework: 10,
+          platform: "windows10-64"
+        }}
+      ]}
```
Comment (klahnakoski) [inline src/quantum/config.js]: @davehunt  Please use the [Files Changed](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/552/files) view to see the code above my comment in full

Above, is the replacement config for MotionMark HTML suite

File: src/quantum/config.js [hunk 4]
```diff
@@ -7,48 +7,149 @@ import { getSignatures } from "../vendor/perfherder";
...
+            'raptor-speedometer-chromium',
+          ],
+          framework: 10,
+          repo: 'mozilla-central',
+        }},
+      ]},
+    },
+
+  ];
+
```
Comment (klahnakoski) [inline src/quantum/config.js]: @davehunt above is the code from `Quantum.jsx`: the config for **Speedometer**.

File: src/quantum/perfherder.jsx [hunk 5]
```diff
@@ -1,283 +0,0 @@
...
-PerfherderWidget.propTypes = {
-  title: PropTypes.string.isRequired,
-  signatures: PropTypes.object,
-  framework: PropTypes.number,
-  explainer: PropTypes.string,
-  reference: PropTypes.string,
-  target: PropTypes.string,
-};
-
-export default withErrorBoundary(PerfherderWidget);
```
Comment (klahnakoski) [inline src/quantum/perfherder.jsx]: @davehunt  We no longer require `perfherder.jsx`  this is just one step of many in an attempt to reduce the total download size of this dashboard:  https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/251

**Comment by davehunt** [comment]
I appreciate the inline comments, thanks @klahnakoski!

**Pull request merged**